Test/FMI3/fmi3_import_model_structure_test.cpp

```diff
@@ -0,0 +1,127 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "fmilib.h"
+#include "config_test.h"
+#include "fmi_testutil.h"
+
+#define CATCH_CONFIG_MAIN
+#include "catch.hpp"
+
+static void test_fmi3_import_get_outputs_list(fmi3_import_t* fmu) {
+
+    fmi3_import_variable_list_t* varList = fmi3_import_get_outputs_list(fmu);
+
+    size_t nVar = 1;
+    REQUIRE(fmi3_import_get_variable_list_size(varList) == nVar);
+
+    fmi3_import_variable_t* var = fmi3_import_get_variable(varList, 0);
+
+    fmi3_value_reference_t vr = 2;
+    REQUIRE(fmi3_import_get_variable_vr(var) == vr);
+
+    fmi3_import_free_variable_list(varList);
+}
+
+static void test_fmi3_import_get_continuous_state_derivatives_list(fmi3_import_t* fmu) {
+
+    fmi3_import_variable_list_t* varList = fmi3_import_get_continuous_state_derivatives_list(fmu);
+
+    size_t nVar = 1;
+    REQUIRE(fmi3_import_get_variable_list_size(varList) == nVar);
+
+    fmi3_import_variable_t* var = fmi3_import_get_variable(varList, 0);
+
+    fmi3_value_reference_t vr = 1;
+    REQUIRE(fmi3_import_get_variable_vr(var) == vr);
+
+    fmi3_import_free_variable_list(varList);
+}
+
+static void test_fmi3_import_get_initial_unknowns_list(fmi3_import_t* fmu) {
+
+    fmi3_import_variable_list_t* varList = fmi3_import_get_initial_unknowns_list(fmu);
+
+    size_t nVar = 1;
+    REQUIRE(fmi3_import_get_variable_list_size(varList) == nVar);
+
+    fmi3_import_variable_t* var = fmi3_import_get_variable(varList, 0);
+
+    fmi3_value_reference_t vr = 1;
+    REQUIRE(fmi3_import_get_variable_vr(var) == vr);
```

Comment:
These kind of variables I'd just inline to make the tests more compact, but up to you. (I know that on test failure you'll get the variable name unexpanded, but it's quite obvious that we're comparing with a VR)

Reply:
Fixed, made the tests a bit more compact.